### PR TITLE
Enhance Docuware Connector

### PR DESCRIPTION
**Pull request by nvvi**
https://1ivy.atlassian.net/browse/MARP-967: Enhance Docuware Connector

1. Fix the variable syntax to make it work with LTS Ivy version
2. Support to put the index field with type Table

### CODE REVIEW

**Review by ntqdinh-axonivy — APPROVED** [review]
LGTM

**Comment by ntqdinh-axonivy** [comment]
Hi @ivy-rew 
Could you have a quick look at this PR since we renamed the variable of  "docuware-connector" to "docuwareConnector" & introduced a new property of "DocuWareFieldTableItem"?
Thank you so much

File: docuware-connector-product/README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Before any interactions between the Axon Ivy Engine and DocuWare services can be
 ```
 Variables:
   
-  docuware-connector:
+  docuwareConnector:
   
     host: <myhost>.docuware.cloud
```
Comment (ivy-rew) [inline docuware-connector-product/README.md:50]: the 'fileCabinetId' is missing here in the readme.
to keep it sync automatically, please use the latest template 'pom.xml' for product and inject the variable listing via @variables.yaml@ as in the newer connectors. 
you may refer to https://github.com/axonivy-market/openai-connector/blob/master/openai-connector-product/README.md?plain=1#L32 for an example

Comment (ntqdinh-axonivy) [reply docuware-connector-product/README.md:50]: Wow, It's a new thing to me. I would handle it right away

**Comment by ivy-rew** [comment]
I think there's still something missing @ntqdinh-axonivy 
the product/pom.xml must be updated in order to actually supply the contents for '@variables.yaml@' . 
you'll need to add the maven-antrun invocation from the latest template to make this work
https://github.com/axonivy-market/market-product/blob/e79cb94fc737a7ea1fe12bff5d0220a0f95a5e30/MY-PRODUCT-NAME-product/pom.xml#L33

**Pull request merged**